### ISSUE TITLE
aquarius url is not correctly configured from env variabler nor configuration file

### ISSUE THREAD

**Issue by jcortejoso** [oceanprotocol/squid-py#141]
# Prerequisites

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed
- [x] I'm reporting the issue to the correct repository.

## Expected Behavior

Squid-py get the address for aquarius host from 1/ Environment variable `AQUARIUS_URL`, and then from config value `aquarius.url`

## Current Behavior

Running environment using `docker-images`, `brizo` tries to connect to Aquarius using always to `localhost:5000`, which fails. `brizo` container has configured the variable `AQUARIUS_URL` and the confiig file `config.ini` with a different value. 

## Failure Information (for bugs)

Please help provide information about the failure if this is a bug. If it is not a bug, please remove the rest of this template.

## Step to Reproduce the problem

Please provide detailed steps for reproducing the issue.

1. Running `docker-images`.
2. Trying to make a request to brizo that needs aquarius.

### Possible Solution

I think the problem is in file [squid_py/config.py](https://github.com/oceanprotocol/squid-py/blob/develop/squid_py/config.py).  During initialization of the `Config` class, the `_load_environ(self)` is only called with `self._section_name = 'keeper-contracts'`, so parameters that are under `self._section_name = resources` are not configured.

**Comment by ttmc** [comment]
In the future, one Brizo instance might be associated with many different Aquarius instances (marketplaces), so it doesn't make sense for there to be one hard-wired AQUARIUS_URL. Brizo doesn't talk directly to Aquarius anyway, does it? As far as I can tell, AQUARIUS_URL could just be removed from the Brizo config settings.

**Comment by ssallam** [comment]
Squid-py requires `AQUARIUS_URL` to be able to connect to aquarius. But this is only necessary if squid-py is to be used for publishing and/or searching assets. Since Brizo only serves consume/compute requests, it does not need aquarius/aquarius_url. Resolving an asset uses the metadata `serviceEndpoint` which is registered on-chain. This is the url that is used in Brizo to resolve an asset did.

In summary, yes, this setting is not required in brizo config or env vars.

The problem reported here is caused by the `aquarius.url` defined in the squid-py instance that published the asset. So the problem description is not accurate and there is no issue in loading the config from envvars or from file.

**Comment by ttmc** [comment]
Related: we've been removing `AQUARIUS_URL` / `aquarius.url` from the Brizo config settings, see:

- https://github.com/oceanprotocol/brizo/issues/43
- https://github.com/oceanprotocol/docker-images/issues/82

**Comment by ssallam** [comment]
This issue seems resolved.
